packages/core/core/src/factories/useCartFactory.ts

```diff
@@ -113,12 +119,16 @@ export const useCartFactory = <CART, CART_ITEM, PRODUCT, COUPON>(
         return;
       }
       loading.value = true;
-      cart.value = await factoryParams.loadCart(context, customQuery);
+      cart.value = await markMethodDeprecated(
+        '\'loadCart\' is deprecated, use \'load\' in your integration instead',
```

Comment:
```suggestion
        '\'loadCart\' is deprecated and will be removed in version 2.1.0, use \'load\' in your integration instead. Read more: https://github.com/DivanteLtd/vue-storefront/pull/5299/',
```

Reply:
please correct all depreciation messages accordingly so the users see
- how long they can use the old api
- some reference to the context of change